django/interface/views.py

```diff
@@ -16,8 +19,108 @@ def statistics(request):
 
 
 def estimate(request):
-    """View of interface estimate"""
-    return render(request, "interface/estimate.html")
+    """View of interface estimate
+    This function calculates the average stay of pets in the shelter based on the
+    get parameters.
+    If there are not enough pets to estimate the range is increased until there are
+    NUMBER_OF_PETS_NEEDED_FOR_ESTIMATE pets. Otherwise error is displayed.
+    If an estimate was calculated some example pets are displayed."""
+
```

Comment:
Django Context dictionary should be crated just before calling django bindings in order to prevent Spagetti code.